spec/setup_acceptance_node.pp

```diff
@@ -12,6 +12,12 @@
   ensure => present,
 }
 
+if $facts['os']['release']['major'] == '7' {
+  package { 'epel-release':
+    ensure => installed,
+  }
+}
+
 if $facts['os']['release']['major'] == '8' {
   # https://tickets.puppetlabs.com/browse/PUP-9978
   exec { '/usr/bin/dnf -y module enable pki-core':
```

Comment:
Unrelated, but now that we've fixed it in candlepin, we can actually remove this